go/core/action.go

```diff
@@ -22,8 +22,8 @@ import (
 
 	"github.com/firebase/genkit/go/core/logger"
 	"github.com/firebase/genkit/go/core/tracing"
-	"github.com/firebase/genkit/go/internal"
 	"github.com/firebase/genkit/go/internal/atype"
+	"github.com/firebase/genkit/go/internal/base"
```

Comment:
"base" and "common" seem equally catch-all. Could we just use a single package?

Reply:
Unfortunately core/tracing imports internal/base, and common imports core/tracing.
Maybe we can come up with a better name for common.